### PR TITLE
Cleanup BTUIKLocalizedString

### PR DESCRIPTION
**Pull request by scannillo**
### Summary of changes

 - Rename `BTUIKLocalizedString` to `BTDropInLocalization`
 - Move methods that don't need to be public to internal header
 - Move localization UI tests from SPMTest app into Demo app

 ### Checklist

 - [X] Added a changelog entry

### Authors
@sestevens @scannillo

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,11 @@
 * Accessibility
   * VoiceOver improvements
   * Fix color contrast ratios
+* Increase hit area for several buttons
```
Comment (sestevens) [inline CHANGELOG.md:8]: Whoops it looks like this line got duplicated. It should be under the "Accessibility" section.
```suggestion
  * Increase hit area for several buttons
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,11 @@
...
+* Localization
+  * Rename `BTUIKLocalizedString` to `BTDropInLocalization`
+  * Remove `insertIntoLocalizedString` methods
+  * Remove translation string static helper methods
   * Increase hit area for several buttons
```
Comment (sestevens) [inline CHANGELOG.md:13]:
```suggestion
```

**Review by sshropshire — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,10 @@
   * VoiceOver improvements
   * Fix color contrast ratios
   * Increase hit area for several buttons
+* Localization
```
Comment (sarahkoop) [inline CHANGELOG.md:9]: Should these be under a Breaking Changes section?

**Review by sarahkoop — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (scannillo) [reply CHANGELOG.md:9]: Good call, yes these are definitely breaking changes

**Pull request merged**